data/DC/incentives.json

```diff
@@ -0,0 +1,330 @@
+  },
+  {
+    "id": "DC-4",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "electric_outdoor_equipment",
+    "program": "dc_dCSustainableEnergyUtility_dCResidentialRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 rebate for electric push lawn mowers."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-5",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "electric_outdoor_equipment",
+    "program": "dc_dCSustainableEnergyUtility_dCResidentialRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$500 rebate for electric riding lawn mowers."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-7",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "dc_dCSustainableEnergyUtility_dCResidentialRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$50 rebate for  ENERGY STAR® qualified smart thermostats."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-9",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "dc_dCSustainableEnergyUtility_dCResidentialRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 700,
+      "minimum": 375,
+      "maximum": 700
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$375-$700 rebate for qualified ducted air source heat pump systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-10",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "dc_dCSustainableEnergyUtility_dCResidentialRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 700,
+      "minimum": 375,
+      "maximum": 700
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$375-$700 rebate for qualified variable speed (ductless) mini-split heat pumps systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-12",
```

Comment:
Incentives DC-12 through DC-20 are odd as currently represented, but I'm not sure what to do about it. It could just be that they changed the program from 2023 to 2024 so the spreadsheet isn't reflecting reality any longer – but if so, the end date should be 2023.

1. I don't see where the any of the numbers/maximums came from.
2. The descriptions should mention that it's for income-qualified residents only, and refer to the financial terms. See comment below for an example of how I'd rewrite it. None of the incentives are tagged with low-income though, which is also an issue.
3. If we're going to split the rows by appliance type, which the spreadsheet has done, the descriptions should mention the appliance type.